tika-dl/pom.xml

```diff
@@ -87,6 +87,11 @@
       <artifactId>nd4j-native-platform</artifactId>
       <version>${dl4j.version}</version>
     </dependency>
+    <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-compress</artifactId>
+        <version>1.14</version>
```

Comment:
commons.compress.version is set in tika-parent's pom.  Reference that here ${commons.compress.version} so we don't have to worry about coordination/conflicts